webapp/graphite/finders/standard.py

```diff
@@ -27,6 +27,7 @@ def find_nodes(self, query):
     clean_pattern = query.pattern.replace('\\', '')
     pattern_parts = clean_pattern.split('.')
 
+    found_leafs, truncated_leafs = 0, 0
```

Comment:
I think could you put both thresholds in intermediary variables, that would make the code easier to read:
truncate_threshold = setings.BLABLA..